Documentation/component-description.md

```diff
@@ -28,7 +32,7 @@ In this page, we focus on the status of each elements. For requirements and desi
     - application/octet-stream. Stream pipeline developer MUST specify the corresponding type and dimensions via properties (input-dim, input-type)
   - You may add subplugins of converters. However, as of 2020-01-10, we do not have any converter subplugins released although we do support them. Users may add such subplugins in run-time.
     - WIP: flatbuffers, protobuf
-- [tensor\_filter](../gst/nnstreamer/tensor_filter)
+- [tensor\_filter](.https://github.com/nnstreamer/nnstreamer/tree/main/gst/nnstreamer/tensor_filter)
```

Comment:
This `.` might be a typo

Reply:
Thanks, I will update after solving CI problem.